docs/installation.rst

```diff
@@ -816,6 +816,84 @@ in this dictionary are made available for users to use in their SQL.
         'my_crazy_macro': lambda x: x*2,
     }
 
+**Scheduling queries**
+
+You can optionally allow your users to schedule queries directly in SQL Lab.
+This is done by addding extra metadata to saved queries, which are then picked
+up by an external scheduled (like [Apache Airflow](https://airflow.apache.org/)).
+
+To allow scheduled queries, add the following to your `config.py`:
+
+.. code-block:: python
+
+    FEATURE_FLAGS = {
```

Comment:
Bumped into an error today where the SQL Lab page fails to render because somehow I set `FEATURE_FLAGS['SCHEDULED_QUERIES']` to `True` in my local configs. 

It is a surprise to me that complex objects are added into "flags". I thought it's only about on/off toggles. This is quite confusing and would make it difficult to extend the `FEATURE_FLAGS` feature structurally (say, moving the management of it to the database or an external API).

Can we maybe introduce a new config value and keep `FEATURE_FLAGS` simple?